### PR TITLE
Community standards

### PR DESCRIPTION
**Pull request by sverhoeven**
Improve https://github.com/eWaterCycle/setup-apptainer/community

Also make README less scary.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+Then run [ncc](https://github.com/zeit/ncc) and push the results:
+```bash
+$ npm run build && npm run package
+$ git add dist
+$ git commit -a -m "prod dependencies"
+```
+
+Create a release on https://github.com/setup-apptainer/setup-apptainer/releases/new with semantic versioning.
+
+Create short tag (`v<number>`) with
```
[the remark this answers is not in the document]
Comment (sverhoeven) [reply CONTRIBUTING.md:104]: Thanks much more inline with GH recommendations

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+$ npm run build && npm run package
+```
+
+### Publish
+
+Actions are run from GitHub repos so we will checkin the packed dist folder. 
+
+Then run [ncc](https://github.com/zeit/ncc) and push the results:
+```bash
+$ npm run build && npm run package
```
[the remark this answers is not in the document]
Comment (sverhoeven) [reply CONTRIBUTING.md]: got rid of ncc as it is a implementation detail, and explained what build and package do

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+$ git add dist
+$ git commit -a -m "prod dependencies"
```
[the remark this answers is not in the document]
Comment (sverhoeven) [reply CONTRIBUTING.md:99]: added

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+```
+
+Create a release on https://github.com/setup-apptainer/setup-apptainer/releases/new with semantic versioning.
+
+Create short tag (`v<number>`) with
+```
+git pull
+git tag -fa v2 -m "Update v2 tag"
+git push origin v2 --force
```
[the remark this answers is not in the document]
Comment (sverhoeven) [reply CONTRIBUTING.md]: Yep, but as we have v2 it also works kind of.

**Comment by sverhoeven** [comment]
> Not sure if you requested a review or wanted me to add to this. Anyway, I left some comments on the contributing guidelines. Also, I would suggest to add a slightly more introductory sentence in the top of the readme. Something like:
> 
> "This action enables you to easily use apptainer images in your GitHub action workflows."

Thanks for reviewing.
I thought the line under the header was an intro, but more intros more better so added your suggestion.

File: CONTRIBUTING.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Peter9192) [reply CONTRIBUTING.md]: I'm not sure I understand. Why have you made v2 point to v1.0.1?

[the remark this answers is not in the document]
Comment (sverhoeven) [reply CONTRIBUTING.md]: I will create a v2.0.0 version so v2 makes sense

**Pull request merged**